plugin.video.retrospect/resources/libs/textures/cached.py

```diff
@@ -0,0 +1,243 @@
+#===============================================================================
+# LICENSE Retrospect-Framework - CC BY-NC-ND
+#===============================================================================
+# This work is licenced under the Creative Commons
+# Attribution-Non-Commercial-No Derivative Works 3.0 Unported License. To view a
+# copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/3.0/
+# or send a letter to Creative Commons, 171 Second Street, Suite 300,
+# San Francisco, California 94105, USA.
+#===============================================================================
+
+import hashlib
+import shutil
+import os
+import io
+
+from backtothefuture import PY2
+from textures import TextureHandler
+
+
+class Cached(TextureHandler):
```

Comment:
you're not allowed to mess with kodi's texture cache nor to download stuff from your own server

Reply:
^^ same as the previous comment.